fv3core/stencils/riem_solver_c.py

```diff
@@ -53,8 +49,11 @@ def finalize(pe2: sd, pem: sd, hs: sd, dz: sd, pef: sd, gz: sd):
         with interval(1, None):
             hs_0 = hs_0[0, 0, -1]
 
-    with computation(PARALLEL), interval(1, None):
-        pef = pe2 + pem
+    with computation(PARALLEL):
+        with interval(0, 1):
+            pef = ptop
+        with interval(1, None):
+            pef = pe2 + pem
```

Comment:
I don't think anyone is focused on refactoring this module at the moment, so the impact here on people should be small and is good to keep.